docs/reference/topology-gc-commandline-reference.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Topology Garbage Collector Cmdline Reference"
+layout: default
+sort: 4
```

Comment:
Manual sorting 🙄, let's put this as the last one
```suggestion
sort: 6
```

Reply:
done